Assets/Scripts/Potions/PotionCrafter.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Written by Trevor West
+// 04/25/2021
+// Class that creates a potion form a list of ingrediants
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PotionCrafter
+{
+    private Potion CreatedPotion;
```

Comment:
This class is kinda lengthy, I wonder if there's a better way to go about this. We're nearing the end so it's not a huge deal.

Reply:
I felt the same way but figured its the end of the semester and doesn't relate to a pattern so just said whatever it works lol